app/components/Blockchain/Transaction/N3ClaimAbstract.jsx

```diff
@@ -0,0 +1,86 @@
+// @flow
+import React from 'react'
+import classNames from 'classnames'
+import { FormattedMessage, IntlShape, injectIntl } from 'react-intl'
+import Button from '../../Button'
+import styles from './Transaction.scss'
+import ClaimIcon from '../../../assets/icons/claim.svg'
+import ContactsAdd from '../../../assets/icons/contacts-add.svg'
+import CopyToClipboard from '../../CopyToClipboard'
+
+type Props = {
+  amount: number,
+  image: string,
+  isPending: boolean,
+  findContact: (address: string) => React$Node | null,
+  from: string,
+  intl: IntlShape,
+  showAddContactModal: (to: string) => void,
+  symbol: string,
+  txDate: React$Node,
+}
+
+class N3ClaimAbstract extends React.Component<Props> {
+  render = () => {
+    const {
+      amount,
+      image,
+      isPending,
+      findContact,
+      from,
+      intl,
+      showAddContactModal,
+      symbol,
+      txDate,
+    } = this.props
+    const logo = image && (
+      <img
+        src={image.replace('ipfs://', 'https://ipfs.io/ipfs/')}
```

Comment:
This logic is in place in other Transfer components to render NFT images, so it can be removed